src/sdk/javascript/examples/functions/events.txt

```diff
@@ -1 +1,7 @@
-{ "subject": "someuserid", "type": "kuehlschrank", "instance": "4711", "feature": "temp", "value": 6 }
```

Comment:
Could you please file an issue? This will impact all examples + their respective configuration + the given events.txt and will be a longer task to resolve.

Reply:
I think we should change it for consistency and understanding but this can be done later. I only want to mention it here.